app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+from math import sqrt, acos, degrees, atan2, sin, cos, radians
+
+
 class Vector:
-    # write your code here
-    pass
+    def __init__(self, coord_x: float, coord_y: float) -> None:
+        self.x = round(coord_x, 2)
+        self.y = round(coord_y, 2)
+
+    def __add__(self, other: Vector) -> Vector:
+        return Vector(
+            coord_x=self.x + other.x,
+            coord_y=self.y + other.y
+        )
+
+    def __sub__(self, other: Vector) -> Vector:
+        return Vector(
+            coord_x=self.x - other.x,
+            coord_y=self.y - other.y
+        )
+
+    def __mul__(self, other: Vector | float) -> Vector | float:
+        if isinstance(other, Vector):
+            return self.x * other.x + self.y * other.y
+        else:
```

Comment:
if it possible try don`t use else operand
here u can move return outside

Reply:
Done